Edit my personal details

bulatovicb opened the pull request.

marijana-codeus commented on app/Http/Controllers/UserController.php:
```diff
@@ -0,0 +1,122 @@
+    {
+        $user = Auth::user();
+        $validator = Validator::make($request->all(), [
+            'first_name' => 'sometimes|required|string',
```
@bulatovicb sometimes i required ne mogu ici zajedno u validaciji

bulatovicb committed:
Summary promjena

Ispravljena greska sa sometimes|required

https://matijabojanic.atlassian.net/browse/OL3-38
